recipes/lorikeet/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "lorikeet" %}
+{% set version = "17" %}
+{% set sha256 = "614e1a8d41dd00b1b92d123c5ca4fa194b4adc7a7dd5f4a796895b86deb08433" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: http://genomeview.org/jenkins/lorikeet/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - openjdk
+  run:
+    - openjdk
+
+test:
+  commands:
+    - lorikeet | grep spoligotype >/dev/null
+
+about:
+  home: https://github.com/AbeelLab/lorikeet
+  summary: 'Tool for digital spoligotyping of MTB strains from Illumina read data'
+  license_family: GPL
+  license: GPL-3
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
```

Comment:
can you please add the DOI to the recipe: 10.1371/journal.pmed.1001880

Reply:
DOI added. Somewhat peculiar - the paper is mentioned on the Lorikeet Github page but there is no mentioned of lorikeet nor of digital spoligotyping in the paper. Still, thanks, it is the correct DOI.